keyboards/crkbd/keymaps/rustyjonez/rules.mk

```diff
@@ -0,0 +1,25 @@
+# Build Options
```

Comment:
You can reduce this file down to:

```make
MOUSEKEY_ENABLE = no
EXTRAKEY_ENABLE = yes
NKRO_ENABLE = yes
RGB_MATRIX_ENABLE = WS2812
SLEEP_LED_ENABLE = yes
OLED_DRIVER_ENABLE = yes
LTO_ENABLE = yes
```

Reply:
I implemented this, but for some reason when I compile using LTO_ENABLE, the firmware size is too large. But when I switched it back to "LINK_TIME_OPTIMIZATION_ENABLE", my firmware compiled just under maximum size at 97%. Other than that the update was made.

Reply:
Ah, your branch might be older than the addition of `LTO_ENABLE` as a shorter alias. You can merge in the upstream master branch to get up to date if you want, but otherwise, don't worry :)